.ci/pipeline/ci.yml

```diff
@@ -324,10 +340,14 @@ jobs:
     vmImage:  'macos-12'
   steps:
   - script: |
-      .ci/scripts/build.sh --platform $(platform.type) --compiler clang --target daal_c --optimizations "sse2 ssse3 sse42 avx2 avx512_mic avx512 avx"
-    displayName: 'make daal_c'
+      brew install dos2unix tree
+      conda create -n ci-env -q -y -c conda-forge openjdk=17.0.3
+    displayName: 'brew and conda install'
+  - script: |
+      .ci/scripts/build.sh --platform $(platform.type) --compiler clang --target daal --optimizations "sse2 ssse3 sse42 avx avx2 avx512" --conda-env ci-env
```

Comment:
Let's keep just sse2, avx2 and avx512 . 

We might need parametrize this and pass to CI. 
1. single arch in review, several for ci, all for nightly